indexing-service/src/main/java/org/apache/druid/indexing/common/task/KillUnusedSegmentsTask.java

```diff
@@ -60,15 +63,23 @@ public class KillUnusedSegmentsTask extends AbstractFixedIntervalTask
 {
   private static final Logger LOG = new Logger(KillUnusedSegmentsTask.class);
 
+  // We split this to try and keep each nuke operation relatively short, in the case that either
```

Comment:
This comment should be moved into a javadoc for the field `batchSize`.

For this constant, add a different comment (preferable javadoc style) instead which sheds some light on why this exact value was chosen.

Reply:
fixed in d3d5a71ddf41dd258a7721b8bd1893b25e83cd11 and 3fc9e96d2b